docs/source/index.rst

```diff
@@ -25,32 +11,9 @@ under study. Utilizing *pyGPCCA*, metastable states as well as cyclic kinetics c
     installation
     api
     example
+    acknowledgements
     references
 
-.. |PyPI| image:: https://img.shields.io/pypi/v/pygpcca
-    :target: https://pypi.org/project/pygpcca/
-    :alt: PyPI
-
-.. |Conda| image:: https://img.shields.io/conda/vn/conda-forge/pygpcca
-    :target: https://anaconda.org/conda-forge/pygpcca
-    :alt: Conda
-
-.. |Cite| image:: https://img.shields.io/badge/DOI-10.1021%2Facs.jctc.8b00079-blue
-    :target: https://doi.org/10.1021/acs.jctc.8b00079
-    :alt: Cite
-
-.. |CI| image:: https://img.shields.io/github/workflow/status/msmdev/pygpcca/CI/main
-    :target: https://github.com/msmdev/pygpcca/actions
-    :alt: CI
-
-.. |Docs|  image:: https://img.shields.io/readthedocs/pygpcca
-    :target: https://pygpcca.readthedocs.io/en/latest
-    :alt: Documentation
-
-.. |Coverage| image:: https://img.shields.io/codecov/c/github/msmdev/pygpcca/main
-    :target: https://codecov.io/gh/msmdev/pygpcca
-    :alt: Coverage
-
 .. |br| raw:: html
```

Comment:
If |br| is no longer used, it can be removed.

Reply:
Done.

Reply:
Was indeed needed, since the beginning of the readme uses |br| and is included in index.rst.